packages/postcss-config/lib/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+const { adminColorSchemes } = require( '@wordpress/base-styles' );
+
+module.exports = {
+	plugins: () => [
```

Comment:
Secondly: Should this package be named as a "configuration"? What does "configuration" mean in the context of PostCSS (not considering Webpack). 

As best I can tell, the only thing PostCSS itself is concerned with for its "constructor" is the plugins:

http://api.postcss.org/postcss.html

Which leads me to believe that the best generalization for shared package would be:

- Named as `@wordpress/postcss-plugins` (`@wordpress/postcss-plugins-preset`? Not sure if there's any convention here in the broader ecosystem).
- No named exported members, only the default export of an array of plugins.

Reply:
Good point. I wasn’t aware that `postcss.config.js` is specific only to webpack loader. It seems like something around preset makes the most sense because those plugins are preconfigured here.

Reply:
`@wordpress/postcss-plugins-preset` it is as of c256fc006845815231289f3d42afe12855ccbc4d.